crates/genie-drs/src/lib.rs

```diff
@@ -171,9 +171,9 @@ impl DRSHeader {
     #[inline]
     /// Read a DRS archive header from a `Read`able handle.
     fn from<R: Read>(source: &mut R) -> Result<DRSHeader, Error> {
-        let mut banner_msg = [0 as u8; 40];
-        let mut version = [0 as u8; 4];
-        let mut password = [0 as u8; 12];
+        let mut banner_msg = [0_u8; 40];
+        let mut version = [0_u8; 4];
+        let mut password = [0_u8; 12];
```

Comment:
is `0_u8` a personal preference over `0u8`?

Reply:
imho it's better visible in the source code with an underscore. and let's number part and type easily being distinguished from one another. not sure if only personal preference though.